ui/imports/shared/popups/SendModal.qml

```diff
@@ -493,39 +492,34 @@ StatusDialog {
 
-                if(gasSelector.suggestedFees.eip1559Enabled && stack.currentGroup === group1 && gasSelector.advancedMode){
-                    if(gasSelector.showPriceLimitWarning || gasSelector.showTipLimitWarning){
-                        Global.openPopup(transactionSettingsConfirmationPopupComponent, {
-                                             currentBaseFee: gasSelector.suggestedFees.baseFee,
-                                             currentMinimumTip: gasSelector.perGasTipLimitFloor,
-                                             currentAverageTip: gasSelector.perGasTipLimitAverage,
-                                             tipLimit: gasSelector.selectedTipLimit,
-                                             suggestedTipLimit: gasSelector.perGasTipLimitFloor,
-                                             priceLimit: gasSelector.selectedOverallLimit,
-                                             suggestedPriceLimit: gasSelector.suggestedFees.baseFee + gasSelector.perGasTipLimitFloor,
-                                             showPriceLimitWarning: gasSelector.showPriceLimitWarning,
-                                             showTipLimitWarning: gasSelector.showTipLimitWarning,
-                                             onConfirm: function(){
-                                                 stack.next();
-                                             }
-                                         })
-                        return
-                    }
+            if(gasSelector.suggestedFees.eip1559Enabled && gasSelector.advancedMode){
+                if(gasSelector.showPriceLimitWarning || gasSelector.showTipLimitWarning){
+                    Global.openPopup(transactionSettingsConfirmationPopupComponent, {
+                                         currentBaseFee: gasSelector.suggestedFees.baseFee,
+                                         currentMinimumTip: gasSelector.perGasTipLimitFloor,
+                                         currentAverageTip: gasSelector.perGasTipLimitAverage,
+                                         tipLimit: gasSelector.selectedTipLimit,
+                                         suggestedTipLimit: gasSelector.perGasTipLimitFloor,
+                                         priceLimit: gasSelector.selectedOverallLimit,
+                                         suggestedPriceLimit: gasSelector.suggestedFees.baseFee + gasSelector.perGasTipLimitFloor,
+                                         showPriceLimitWarning: gasSelector.showPriceLimitWarning,
+                                         showTipLimitWarning: gasSelector.showTipLimitWarning,
+                                         onConfirm: function(){
+                                             stack.currentIndex = 1
+                                         }
+                                     })
+                    return
                 }
-
-                stack.next()
             }
+            stack.currentIndex = 1
```

Comment:
Could we add a constant and/or enum for the stack panels. That way, we know what `1` is